config/config.txt

```diff
@@ -1,35 +1,35 @@
-## Server name: This appears at the top of the screen in-game and in the BYOND hub. Remove the # infront of SERVERNAME and replace 'tgstation' with the name of your choice.
+## Server name: This appears at the top of the screen in-game and in the BYOND hub. Activate and replace 'tgstation' with the name of your choice.
```

Comment:
The issue I'm trying to solve is that the tool infers documentation from the original versions of these files, like so: 
![untitled](https://cloud.githubusercontent.com/assets/8171642/26435663/da94ec28-40de-11e7-9546-88d821fa7f9d.png)